tests/ios/MASVS-PLATFORM/MASTG-TEST-0072.md

```diff
@@ -51,3 +51,4 @@ ph.telegra.Telegraph on (iPhone: 11.1.2) [usb] # cd PlugIns
 
 ph.telegra.Telegraph on (iPhone: 11.1.2) [usb] # ls
 NSFileType      Perms  NSFileProtection    Read    Write     Name
+```
```

Comment:
Please go back to v1.5.0 and include the missing text from Document/0x06h-Testing-Platform-Interaction.md

Adapt the heading accordingly.

```suggestion
You can also access per SSH, ...

#### Dynamic Analysis

For the dynamic analysis we can do the following to gain knowledge without having the source code:

- Inspecting the items being shared
- Identifying the app extensions involved
```